pkg/telemetry/prometheus/packets.go

```diff
@@ -56,6 +58,8 @@ var (
 	promRTT             *prometheus.HistogramVec
 	promParticipantJoin *prometheus.CounterVec
 	promConnections     *prometheus.GaugeVec
+	promForwardLatency  prometheus.Gauge
+	promForwardJitter   prometheus.Gauge
```

Comment:
Should we make these histograms so that we can do things like `p50, p99` etc?

Reply:
The latency & jitter here is a smoothed value here so use gauge to observe the pattern of latency for each node. If change it to histograms then we need to report it for each forwarding packet there will have a performance impact in heavy load?

Reply:
ah got it, thank you @cnderrauber